features/karaf/openhab-core/src/main/feature/feature.xml

```diff
@@ -39,7 +39,6 @@
 		<bundle>mvn:org.openhab.core.bundles/org.openhab.core.config.dispatch/${project.version}</bundle>
 		<bundle start-level="75">mvn:org.openhab.core.bundles/org.openhab.core.config.xml/${project.version}</bundle>
 		<bundle>mvn:org.openhab.core.bundles/org.openhab.core/${project.version}</bundle>
-		<feature dependency="true">openhab-core-storage-mapdb</feature>
```

Comment:
Using `openhab-core-storage-json` here instead of removing the dependency will probably fix it.